services/core-web/src/components/parties/RelationshipProfile.tsx

```diff
@@ -0,0 +1,286 @@
+      : [];
+
+  const columns: ColumnsType<ITransformedValues> = [
+    {
+      title: "Contact",
+      dataIndex: "contact",
+      render: (text, record) => (
+        <div title="Contact">
+          <Link to={router.PARTY_PROFILE.dynamicRoute(record.partyGuid)}>{text}</Link>
+        </div>
+      ),
+    },
+    renderTextColumn("role", "Role"),
+    ...permitColumn,
+    ...EORColumn,
+    {
+      title: "Dates",
+      dataIndex: "dates",
+      render: (text, record) => (
+        <Row title="Dates" align="middle" justify="space-between">
+          <Col>
+            {record.startDate} - {record.endDate}
+          </Col>
+          {canEdit && (
+            <Col>
+              <CoreButton type="text" onClick={() => handleOpenEditPartyRelationshipModal(record)}>
+                Edit
+              </CoreButton>
+            </Col>
+          )}
+        </Row>
+      ),
+    },
+  ];
+
+  const transformRowData = (historyRelationships: IPartyAppt[]): ITransformedValues[] =>
+    historyRelationships.map((relationship) => {
+      const partyRelationshipType = partyRelationshipTypes.find(
+        (r) => r.value === relationship.mine_party_appt_type_code
+      ).label;
+      const partyRelationshipTitle = `Update ${partyRelationshipType}: ${relationship.party.name}`;
+      return {
+        key: relationship.mine_party_appt_guid,
+        contact: relationship.party.name,
+        partyGuid: relationship.party.party_guid,
+        partyRelationship: relationship,
+        role: partyRelationshipTitle,
+        permit: permitsMapping[relationship.related_guid],
+        tailingsStorageFacility: TSFMapping[relationship.related_guid],
+        endDate: formatDate(relationship.end_date) || "Present",
+        startDate: formatDate(relationship.start_date) || "Unknown",
+      };
+    });
+
+  const filteredRelationships: IPartyAppt[] = partyRelationships
+    .sort((a, b) =>
+      moment(a.start_date, "YYYY-MM-DD") >= moment(b.start_date, "YYYY-MM-DD") ? -1 : 1
+    )
+    .filter(({ mine_party_appt_type_code }) => mine_party_appt_type_code === typeCode);
+
+  if (isLoaded) {
+    return (
+      <div className="profile">
+        <div className="profile__header">
+          <div className="inline-flex between">
+            <h1 className="bold">{partyRelationshipTitle} History</h1>
+          </div>
+          <div className="inline-flex between">
+            <div className="inline-flex">
+              <p>
+                <Link to={router.MINE_CONTACTS.dynamicRoute(mine.mine_guid)}>
+                  {mine && mine.mine_name}
+                </Link>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="profile__content">
+          <Tabs
+            activeKey="history"
+            size="large"
+            animated={{ inkBar: true, tabPane: false }}
+            centered
+          >
+            <Tabs.TabPane tab="History" key="history">
```

Comment:
not a big deal- but we get console errors/warnings about Tabs.TabPane being deprecated, I think it would be really easy to convert this one to `<Tabs items={tabItems} />` with just the one tab.

Reply:
Good call!